aiida/cmdline/commands/cmd_status.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida_core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""`verdi profile` command."""
+from __future__ import division
+from __future__ import print_function
+from __future__ import absolute_import
+
+from enum import IntEnum
+import click
+
+from aiida.cmdline.commands.cmd_verdi import verdi
+from aiida.cmdline.utils.daemon import get_daemon_status
+from aiida.manage import get_manager
+from aiida.common.utils import Capturing, get_repository_folder
+from aiida.manage.external.rmq import get_rmq_url
+from aiida.manage.external.postgres import Postgres
+
+
+class ServiceStatus(IntEnum):
+    """Describe status of services for 'verdi status' command.
+    """
+    UP = 0  # pylint: disable=invalid-name
+    ERROR = 1
+    DOWN = 2
+
+
+STATUS_SYMBOLS = {
+    ServiceStatus.UP: {
+        'color': 'green',
+        'string': u'\u2713',
+    },
+    ServiceStatus.ERROR: {
+        'color': 'red',
+        'string': u'\u2717',
+    },
+    ServiceStatus.DOWN: {
+        'color': 'red',
+        'string': u'\u2717',
+    },
+}
+
+
+@verdi.command('status')
+def verdi_status():
+    """Print status of AiiDA services."""
+    # pylint: disable=broad-except
+
+    manager = get_manager()
+
+    # getting the profile
+    try:
+        profile = manager.get_profile()
+        print_status(ServiceStatus.UP, 'profile', "On profile {}".format(profile.name))
+
+    except Exception as exc:
+        print_status(ServiceStatus.ERROR, 'profile', "Unable to read AiiDA profile")
+        raise exc  # without a profile, we cannot access anything
+
+    # getting the repository
+    try:
+        repo_folder = get_repository_folder()
+    except Exception as exc:
+        print_status(ServiceStatus.ERROR, 'repository', "Error with repo folder")
+        print(exc)
```

Comment:
Maybe its better to use `click` to print the exception like e.g. `click.echo(exc, err=True)`